drivers/hmis_csv_importer/app/views/hmis_csv_importer/imported/show.haml

```diff
@@ -13,11 +13,13 @@
           %tr
             - @klass.hmis_structure.each_key do |header|
               %th= header
+          %th Expired
         %tbody
           - @data.each do |row|
             %tr
               - row.hmis_data.each do |key, value|
                 %td= link_to_if key.to_sym == @klass.hud_key, value, source_data_path(search: {type: entry_name, id: value, data_source_id: row.data_source_id })
+              %td= row.expired ? 'Y' : 'N'
```

Comment:
We have a helper for this:
```suggestion
              %td= yes_no(row.expired)
```
or if you want to get fancy, we have `checkmark_or_x`